lib/cosmos/top_level.rb

```diff
@@ -606,13 +606,15 @@ def self.require_class(class_filename)
   # Requires a file with a standard error message if it fails
   #
   # @param filename [String] The name of the file to require
-  def self.require_file(filename)
+  # @param log_error [Boolean] Whether to log an error if the require fails
+  def self.require_file(filename, log_error = true)
     begin
       require filename
-    rescue Exception => err
+    rescue LoadError => err
```

Comment:
Lots of more things can go wrong here beside load errors.   Need to catch Exception.   LoadError is only if the file is not found.  Could have syntax errors and all kinds of other problems we want to catch too.